maps/away/ships/tirakqi_smuggler/tirakqi_smuggler.dm

```diff
@@ -5,7 +5,7 @@
 	prefix = "ships/tirakqi_smuggler/"
 	suffixes = list("tirakqi_smuggler.dmm")
 
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS)
+	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS, SECTOR_UUEOAESA)
```

Comment:
why not all of badlands at this point

Reply:
that should be fine to do i just didnt know that tag existed